six/modules/c++/six.sidd/include/six/sidd/SFA.h

```diff
@@ -261,6 +355,16 @@ struct SFAMultiPoint : public SFAGeometryCollection
     }
     std::vector<mem::ScopedCopyablePtr<SFAPoint> > vertices;
     static const char TYPE_NAME[];
+
+    virtual bool equalTo(const SFATyped& rhs) const
+    {
+        SFAMultiPoint const* multiPoint = dynamic_cast<SFAMultiPoint const*>(&rhs);
```

Comment:
Oh also with all of these, style-wise the `const` is in the wrong spot.  It should be...

`const SFAMultiPoint* multiPoint = dynamic_cast<const SFAMultiPoint*>(&rhs);`